src/PKSim.Core/Model/AdvancedParameter.cs

```diff
@@ -66,13 +66,13 @@ public double GetPercentileForValue(double value)
       ///    Generates an array of random values. The dimension of the array is equal to numberOfValues
       /// </summary>
       /// <param name="numberOfValues">Number of values to generate</param>
-      public virtual IEnumerable<RandomValue> GenerateRandomValues(int numberOfValues)
+      public virtual IReadOnlyList<RandomValue> GenerateRandomValues(int numberOfValues)
       {
          ResetGenerator();
```

Comment:
I am not sure if this makes sense or not. That means that a given adavned parameter will always create the same series of random values...
This makes however our use case possible

Reply:
If we do not reset, there is NO way we can recreate the same population parameters.
I think this is absolutely ok when you think about it, because EACH advanced parameter will have a different seed to they are independent

Reply:
and remove the advanced parameter and added it again will create another seed